### DISCUSSION TITLE
¿Hay intención de seguir con las actualizaciones?

### DISCUSSION THREAD

**Discussion by SruiSan** [gajosu/ha-whatsapp-web-rest-api#179] [General]
Este proyecto es superinteresante y muy potente para la interconexión de HA con Whatsapp. Supongo que por cuestión de tiempo no está siendo posible mantenerlo, pero ¿hay esperanza de actualización próxima?

Muchas Gracias.

**Comment by gajosu** [comment]
Hola! Ayer hice una nueva update que fixea los problemas que habían, pruebala y me dices si todo te va bien!

**Reply by SruiSan** [reply]
Por ahora no funciona. No se puede acceder por consola al contenedor y en el log se muestra:

10/3/2024, 14:48:48 [info]: 	Starting Controller 
10/3/2024, 14:48:48 [info]: 	[WebSocket] Starting WebSocket 
10/3/2024, 14:48:48 [info]: 	[HttpServer] ⚡ Listening http://0.0.0.0:3000 

Pero no llega ni siquiera a generar el QR. Se borró la carpeta data y se regeneró el docker, pero el resultado fue el mismo.

**Comment by gajosu** [comment]
fixed in v2.4.1

**Comment by SruiSan** [comment]
Sigue sin arrancar:

12/2/2024, 21:44:41 [info]: 	Starting Controller 
12/2/2024, 21:44:41 [info]: 	[WebSocket] Starting WebSocket 
12/2/2024, 21:44:41 [info]: 	[HttpServer] ⚡ Listening http://0.0.0.0:3000 
Error: Could not find expected browser (chrome) locally. Run `npm install` to download the correct Chromium revision (1045629).
  at PuppeteerNode.launch (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/node/PuppeteerNode.js:151:31)
